Mathlib/Analysis/Convex/Topology.lean

```diff
@@ -69,6 +69,9 @@ theorem isCompact_stdSimplex : IsCompact (stdSimplex ℝ ι) :=
   Metric.isCompact_iff_isClosed_bounded.2 ⟨isClosed_stdSimplex ι, bounded_stdSimplex ι⟩
 #align is_compact_std_simplex isCompact_stdSimplex
 
+instance stdSimplex.instCompactSpace : CompactSpace (stdSimplex ℝ ι) :=
```

Comment:
Just to make it a little more obvious why we have both:
```suggestion
instance stdSimplex.instCompactSpace_coe : CompactSpace ↥(stdSimplex ℝ ι) :=
```
(which is obviously syntactically the same)